### PR TITLE
[WIP] Make metrics E2E test work in PROW

### PR DESCRIPTION
**Pull request by nachocano**
Fixes https://github.com/google/knative-gcp/issues/317

### CODE REVIEW

**Comment by nachocano** [comment]
let's see if this fixes it

**Comment by nachocano** [comment]
trying one more time with @chizhg suggestion in https://github.com/google/knative-gcp/issues/317

File: test/e2e-common.sh [hunk 1]
```diff
@@ -76,6 +83,7 @@ function pubsub_setup() {
   if (( ! IS_PROW )); then
     echo "Set up ServiceAccount for Pub/Sub Admin"
     gcloud services enable pubsub.googleapis.com
+    gcloud services enable monitoring
```
Comment (chaodaiG) [inline test/e2e-common.sh]: move this line to be above line 83

Comment (nachocano) [reply test/e2e-common.sh]: done!

**Comment by nachocano** [comment]
@chaodaiG @chizhg I made the change but still no luck.. is weird... If you guys have access to the UI, we can check the metrics explorer under monitoring to see if they are actually sent.
I can add a big sleep so that we can login in the cluster, and see the logs from the pods as well...
will ping you back next week, thanks!

**Comment by nachocano** [comment]
will continue on another one

**Pull request closed without merging**